apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/SkippedRifRecord.java

```diff
@@ -39,7 +39,7 @@ public class SkippedRifRecord {
       generator = "skipped_rif_records_record_id_seq")
   @SequenceGenerator(
       name = "skipped_rif_records_record_id_seq",
-      sequenceName = "skipped_rif_records_record_id_seq",
+      sequenceName = "ccw.skipped_rif_records_record_id_seq",
```

Comment:
I found one more sequence reference here: https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/resources/GenerateEntitiesFromDslMojoIT/expected/FissClaim.java#L200 does this need to be updated?

Reply:
It's test data. Nothing to see here.